### ISSUE TITLE
[Bug] Geo3k multi turn reward abnormal

### ISSUE THREAD

**Issue by Zhou-jiecheng** [THUDM/slime#1724]
### Bug Description

I'm trying to reproduce the reward curve of examples/geo3k_multi_turn, but I failed; the rewards do not increase. Even the first step raw reward is around 0.1, which is expected to be around 0.3.....

### Steps to Reproduce

1. use the official Docker container;
2. run examples/geo3k_multi_turn training script.

### Expected Behavior

The first step raw reward is around 0.3

### Actual Behavior

The first step raw reward is around 0.1

### Environment

- slime version:
- Python version:
- PyTorch version:
- CUDA/ROCm version:
- GPU type and count:
- OS:
- SGLang version (if relevant):
- Megatron-LM version (if relevant):

### Logs

```shell

```

### Pre-submission Checklist

- [x] I have read the [CONTRIBUTING.md](https://github.com/THUDM/slime/blob/main/CONTRIBUTING.md) and understand the collaboration scope.
- [x] I have read the [documentation](https://thudm.github.io/slime/) and my issue is not addressed there.
- [x] I have searched for [existing issues](https://github.com/THUDM/slime/issues) and this is not a duplicate.
- [x] I have provided a minimal, reproducible example.

**Comment by ZHZisZZ** [comment]
https://github.com/THUDM/slime/pull/1727

This issue is probably caused by a bug in SGLang 0.5.9 that leaves the Qwen3-VL vision encoder unloaded.

**Comment by Zhou-jiecheng** [comment]
> [#1727](https://github.com/THUDM/slime/pull/1727)
> 
> This issue is probably caused by a bug in SGLang 0.5.9 that leaves the Qwen3-VL vision encoder unloaded.

Let me try.

**Comment by Zhou-jiecheng** [comment]
It does not work for me, I switched to docker slime v0.2.2 and worked

**Comment by Alzheimer-hz** [comment]
Same problem, I pull the latest docker on Qwen3-VL-2B-Instrcuct. SGLang seems to have much lower performance on geo3k test
  ┌─────────────┬──────────┬──────┐
  │   Server    │ Accuracy │ Time │
  ├─────────────┼──────────┼──────┤
  │ vLLM TP=8   │ 55.0%    │ 686s │
  ├─────────────┼──────────┼──────┤
  │ SGLang DP=8 │ 10.0%    │ 158s │
  └─────────────┴──────────┴──────┘

**Comment by ZHZisZZ** [comment]
@Alzheimer-hz 

Have you had a chance to try this PR?
https://github.com/THUDM/slime/pull/1727

I suspect it might be related to the issue. I’d appreciate it if you could help verify.

**Comment by Alzheimer-hz** [comment]
I have tried and it worked
